packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,41 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+<div class="pt-callout pt-intent-success pt-icon-comparison">
+  <h5>Simple labels vs. form groups</h5>
+  <p>Blueprint provides two ways of connecting label text to control fields, depending on the complexity of the control.</p>
+  <p>Simple labels are a basic way to connect a label with a single control.</p>
+  <p>Form groups require more markup to maintain consistent visuals across more complex controls.</p>
```

Comment:
put the reason first:

"Form groups support more complex control layouts but require more markup to maintain consistent visuals."